oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/InfluxClient.java

```diff
@@ -205,4 +230,20 @@ public static TimeInterval timeIntervalTS(long timestamp) {
     public static TimeInterval timeIntervalTB(long timeBucket) {
         return ti(TimeBucket.getTimestamp(timeBucket), "ms");
     }
+
+    @Override
+    public void registerChecker(HealthChecker healthChecker) {
+        this.healthChecker.register(healthChecker);
+    }
+
+    /**
+     * Check influx health, Ignore grammar related exception
+     */
+    private void checkHealth(Throwable e) {
+        if (e instanceof InfluxDBIOException) {
+            healthChecker.unHealth(e);
+        } else if (!(e instanceof InfluxDBException)) {
+            healthChecker.unHealth(e);
+        }
```

Comment:
Grammar related exceptions seem `unhealth` to me. @hanahmily What do you think?

Reply:
done